brave/brave-core
# Refactor loadFromJson from RapidJSON to base::JSONReader

Fixes https://github.com/brave/brave-browser/issues/6351

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm ledger state is persisted for upgrade paths
- Confirm ledger state is persisted for fresh installs

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
